src/LendingPool.sol

```diff
@@ -97,33 +104,48 @@ contract LendingPool is Guardian, TrustedCreditor, DebtToken, InterestRateModule
     /**
      * @notice Adds a tranche to the Lending Pool
      * @param tranche The address of the Tranche
-     * @param _weight The weight of the specific Tranche
+     * @param _interestWeight The interestWeight of the specific Tranche
+     * @param liquidationWeight The liquidationWeight of the specific Tranche
      * @dev The order of the tranches is important, the most senior tranche is at index 0, the most junior at the last index.
      * @dev Each Tranche is an ERC-4626 contract
-     * @dev The weight of each Tranche determines the relative share yield (interest payments) that goes to its Liquidity providers
-     * @dev ToDo: For now manually add newly created tranche, do via factory in future?
+     * @dev The interestWeight of each Tranche determines the relative share yield (interest payments) that goes to its Liquidity providers
      */
-    function addTranche(address tranche, uint256 _weight) public onlyOwner {
+    function addTranche(address tranche, uint16 _interestWeight, uint16 liquidationWeight) public onlyOwner {
```

Comment:
Variables should be underscore suffix not prefix

Reply:
variable can be removed here, original interestWeight has been removed